tests/unit/raid_abstraction/test_megaraid.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+
+import mock
+
+from mercury_agent.hardware.drivers.megaraid import MegaRAIDActions
+
+from ..base import MercuryAgentUnitTest
+
+
+def get_storcli_dall_show(controller):
+    assert controller is not None
```

Comment:
Noting this is not enforced if debugging is not activated. A test/raise would be a for sure hard stop.

Reply:
This is a stub test case that I used for debugging. The final test case will be more complete :)